numerapi/numerapi.py

```diff
@@ -6,10 +6,17 @@
 import datetime
 import decimal
 from typing import List, Dict
+from io import BytesIO
 
 # Third Party
 import requests
 import pytz
+import pandas as pd
```

Comment:
pandas is a new dependency. It needs to be added in `setup.py` and in `requirements.txt`